### PR TITLE
wavespectra package added

### PR DESCRIPTION
**Pull request by RubendeBruin**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by RubendeBruin** [comment]
@rafa-guedes Can you confirm that you're okay with being listed as maintainer for this package? I will do the work, but need to be able to control this package as well in just in case

**Comment by rafa-guedes** [comment]
Yes sure @RubendeBruin

**Comment by RubendeBruin** [comment]
@conda-forge/help-python Package is ready to be merged.

File: recipes/wavespectra/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8ae6cf102ff3ef9bfb94bbc5842533746336489a97d9233cee5ba7f0f60681d1
+  
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
```
Comment (chrisburr) [inline recipes/wavespectra/meta.yaml:14]: The comments from the example should be removed

File: recipes/wavespectra/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    - wavespectra
+
+about:
+  home: https://github.com/wavespectra/wavespectra
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Python library for ocean wave spectra'
+  description: |
+    Wavespectra is an open source project for working with ocean wave spectral data. The library is built on top of xarray, leveraging from xarray’s labelled multi-dimensional arrays and making dealing with wave spectra simple and fast..
```
Comment (chrisburr) [inline recipes/wavespectra/meta.yaml:79]: This should be wrapped to ~80 characters

File: recipes/wavespectra/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('fortran') }}
+    - numpy
```
Comment (chrisburr) [inline recipes/wavespectra/meta.yaml:28]:
```suggestion
```

File: recipes/wavespectra/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    - pandas
+    - python-dateutil
+    - pyyaml
+    - sortedcontainers
+    - scipy
+    - toolz
+    - xarray
+    - zarr
+    - pytest-runner
```
Comment (chrisburr) [inline recipes/wavespectra/meta.yaml:48]: It's likely that many of these aren't needed in `host`. I would start from `numpy`, `pytest-runner`, `python` and `pip` and expand from there.

Comment (RubendeBruin) [reply recipes/wavespectra/meta.yaml]: @chrisburr , thanks for your review.
Didn't want to abuse your system with many trials-and-error runs. I guessed that when pytest-runner is used then some of the pytests will be run which would require the packages.

Comment (chrisburr) [reply recipes/wavespectra/meta.yaml]: IIRC `pytest-runner` is just used to make `python setup.py test` work. It definitely doesn't cause the unit tests to be executed.

As `pytest-runner` is included in `setup_requires` of the `setup.py` it is actually needed even if it doesn't do anything.

File: recipes/wavespectra/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    - sortedcontainers
+    - scipy
+    - toolz
+    - xarray
+    - zarr
+
+test:
+  # Only test if the module can be imported. We do not want to bother conda-forge with the whole test suite (yet)
+  imports:
+    - wavespectra
```
Comment (chrisburr) [inline recipes/wavespectra/meta.yaml:47]:
```suggestion
    - wavespectra
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/wavespectra/meta.yaml (hunk 4 above)
Comment (RubendeBruin) [reply recipes/wavespectra/meta.yaml]: ok, thanks for explaining. I removed most of the packages. Indeed only pytest-runner was required.

**Comment by RubendeBruin** [comment]
@chrisburr ready to be merged (I hope)

**Comment by RubendeBruin** [comment]
@chrisburr gentle nudge...

**Comment by RubendeBruin** [comment]
@conda-forge/staged-recipes Gentle reminder: Ready to be merged

**Comment by RubendeBruin** [comment]
@conda-forge/staged-recipes : Updated recipe to new release of the package. Ready to be merged.

**Pull request merged**